nntrainer/tensor/cache_pool.cpp

```diff
@@ -65,7 +65,7 @@ convertTensorLifespanToCachePolicy(const TensorLifespan lifespan) {
     policy = CachePolicy::ITERATION_CONSIST;
     break;
   case TensorLifespan::FORWARD_GRAD_AGRAD_LIFESPAN:
-    policy = CachePolicy::ITERATION_CONSIST;
+    policy = CachePolicy::ALWAYS_SYNCED;
```

Comment:
LGTM.

However, if this inline function is access frequently, consider refactoring it as an array:

```
static const CachePolicy policy[TensorLifespan::MAX] = {
  [UNMANAGED] = ALWAYS_SYNCED,
  ...
};
```

Reply:
Thanks. I'll modify it.

Reply:
I checked the array initializer you mentioned, but it does not be supported  in c++17(currently used)
```
sorry, unimplemented: non-trivial designated initializers not supported
```
The designated initializer is supported after c++20.
https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/p0329r4.pdf

I think it's not the time to apply the new standard yet.